deploy/crds/pxc.percona.com_perconaxtradbclusterbackups_crd.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Why crds dir needed&

Reply:
The `crds` directory was automatically generated by osdk and I mistakenly assumed that the path was correct.
Now I have merged the generated CRDs into the `deploy/crd.yaml`, in the hope that they are now correctly stored there?

Reply:
My initial question was about why do we even need all these new CRDs.
We have already had it.
I can't understand how it's related to fixing of usage of deprecated packages

Reply:
The CRD generation is not necessary for the actual fix. I made all these changes just because of this prompt:

> But your code is not building. Please fix all issues and commit fix =)

So, even though my initial change shouldn't actually cause any build problems, I tried to build the operator locally. However, I succeeded only after all the changes here. Among them was the change to the types and I had the CRDs regenerated to fit the changed types.